File: 131. 프린터(js)/printer.js
function solution(priorities, location) {
  let rank = 0;
  while (1) {
    if (priorities[0] === Math.max.apply(null, priorities)) {
      priorities.splice(0, 1);
      rank += 1;
      if (location === 0) {
        return rank;
      } else {
        location -= 1;
      }
    } else {
      priorities = [...priorities.slice(1), priorities[0]];
      location = location === 0 ? priorities.length - 1 : location - 1;
    }
  }
}

File: Python/053. 프린터/Printer.py
def solution(priorities, location):
    answer = 0
    tmp = [i for i in range(len(priorities))]
    while True:
        num = priorities.pop(0)
        if num < max(priorities):
            priorities.append(num)
            tmp.append(tmp.pop(0))
        else:
            answer += 1
            if tmp.pop(0) == location:
                break
    return answer